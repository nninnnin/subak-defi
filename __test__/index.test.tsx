import { render, screen } from "@testing-library/react";
import Home from "../src/pages/index";
import "@testing-library/jest-dom";

describe("Home", () => {
  it("renders a container", async () => {
    render(<Home />);

    const container = screen.getByRole("container");

    expect(container).toBeInTheDocument();
  });

  it("renders container unchanged", () => {
    const { container } = render(<Home />);

    expect(container).toMatchSnapshot();
  });
});
