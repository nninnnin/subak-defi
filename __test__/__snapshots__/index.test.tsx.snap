// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders container unchanged 1`] = `
<div>
  <div
    role="container"
  >
    index2
  </div>
</div>
`;
